### PR TITLE
UpdateContact Method is updating all numbers in the contact instead of a specific number is fixed.

### PR DESCRIPTION
**Pull request by AnkitKaneri**
#287

### CODE REVIEW

File: android/src/main/java/com/rt2zz/reactnativecontacts/ContactsManager.java [hunk 1]
```diff
@@ -540,7 +566,7 @@ private void requestReadContactsPermission(Callback callback) {
     }
 
     protected static void onRequestPermissionsResult(int requestCode, @NonNull String[] permissions,
-        @NonNull int[] grantResults) {
+                                                     @NonNull int[] grantResults) {
```
[the remark this answers is not in the document]
Comment (AnkitKaneri) [reply android/src/main/java/com/rt2zz/reactnativecontacts/ContactsManager.java]: This is done by android studio's code formatter. Do you want me to remove these? 
ref : http://www.oracle.com/technetwork/java/codeconventions-150003.pdf 
section : 4.2 Wrapping Lines

File: android/src/main/java/com/rt2zz/reactnativecontacts/ContactsManager.java [hunk 2]
```diff
@@ -90,11 +88,13 @@ public void run() {
...
         getAllContactsMatchingString(searchString, callback);
     }
+
     /**
      * Retrieves contacts matching String.
      * Uses raw URI when <code>rawUri</code> is <code>true</code>, makes assets copy otherwise.
+     *
      * @param searchString String to match
-     * @param callback user provided callback to run at completion
+     * @param callback     user provided callback to run at completion
```
[the remark this answers is not in the document]
Comment (AnkitKaneri) [reply android/src/main/java/com/rt2zz/reactnativecontacts/ContactsManager.java]: done.

File: android/src/main/java/com/rt2zz/reactnativecontacts/ContactsManager.java [hunk 3]
```diff
@@ -112,8 +112,9 @@ public void run() {
 
     /**
      * Retrieves <code>thumbnailPath</code> for contact, or <code>null</code> if not available.
+     *
      * @param contactId contact identifier, <code>recordID</code>
-     * @param callback callback
+     * @param callback  callback
```
[the remark this answers is not in the document]
Comment (AnkitKaneri) [reply android/src/main/java/com/rt2zz/reactnativecontacts/ContactsManager.java]: done.

File: android/src/main/java/com/rt2zz/reactnativecontacts/ContactsManager.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (morenoh149) [reply android/src/main/java/com/rt2zz/reactnativecontacts/ContactsManager.java]: hmm, if it's android studio default then that is fine.

**Comment by morenoh149** [comment]
thank you @AnkitKaneri

**Comment by AnkitKaneri** [comment]
My pleasure 😄

**Pull request merged**